src/colorpickr.js

```diff
@@ -237,8 +249,12 @@ class ColorPickr extends React.Component {
     }
 
     const rgbaBackground = rgbaColor(r, g, b, a);
-    const opacityGradient =
-      `linear-gradient(to right, ${rgbaColor(r, g, b, 0)}, ${rgbaColor(r, g, b, 100)})`;
+    const opacityGradient = `linear-gradient(to right, ${rgbaColor(
+      r,
+      g,
+      b,
+      0
```

Comment:
This formatting is funny. Can you force Prettier to break the next rgbaColor call into multiple lines?